docs/cooperation/cooperation.yaml

```diff
@@ -0,0 +1,304 @@
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 403
+                code: FORBIDDEN
+                message: You do not have permission to perform this action.
+  /cooperations/{id}:
+    get:
+      security:
+        - bearerAuth: [ ]
+      tags:
+        - Cooperations
+      summary: Find cooperation by ID
+      description: Finds and returns cooperations with the specified ID.
+      produces:
+        - application/json
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: ID of the cooperation that needs to be fetched
+          type: string
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/definitions/cooperationResponse'
+              example:
+                _id: 8755bc080a00adr9243df106
+                offerId: 64ebc6fbd2f34037d0aba782
+                tutorId: 6755bc080a75adf9223df210
+                studentId: 6655bc080a75adf9223df108
+                price: 150
+                status: pending
+                createdAt: 2022-10-18T13:25:36.292+00:00
+                updatedAt: 2022-10-18T13:25:36.292+00:00
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 400
+                code: INVALID_ID
+                message: ID is invalid.
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 401
+                code: UNAUTHORIZED
+                message: The requested URL requires user authorization.
+        404:
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 404
+                code: COOPERATION_NOT_FOUND
+                message: Cooperation with the specified id was not found.
+    patch:
+      security:
+        - bearerAuth: [ ]
+      tags:
+        - Cooperations
+      summary: Update cooperation by ID
+      description: Finds and update a cooperation with the specified ID.
+      produces:
+        - application/json
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: Cooperation ID
+          type: string
+      requestBody:
+        required: true
+        description: Provide required data to update status of cooperation.
+        content:
+          application/json:
+            schema:
+              $ref: '#/definitions/cooperation'
+            example:
```

Comment:
remove offerId tutorId studentId
i am pretty sure only price and status should be available for the update